pom.xml

```diff
@@ -220,7 +220,7 @@
                                     <property name="format" value="^_?[A-Z][a-zA-Z0-9]*$|packageinfo" />
                                 </module>
                                 <module name="AvoidStarImport">
-                                    <property name="excludes" value="java.io,java.net,java.util,javax.enterprise.inject.spi,javax.enterprise.context" />
+                                    <property name="excludes" value="java.io,java.net,java.util,jakarta.enterprise.inject.spi,jakarta.enterprise.context" />
```

Comment:
I think this needs to stay as “javax” since it is a package name.  The packages don’t change until Jakarta 9, just the maven coordinates.